test/docker/executeContainerCommand.test.js

```diff
@@ -4,6 +4,7 @@ const { executeContainerCommand } = require('../../src/docker');
 
 test('response includes stdout, stderr, and inspectOutput with ExitCode 0', async (t) => {
   const container = await createDefaultContainer();
+  console.log(`Started container ${container.id}`);
```

Comment:
Did you mean to make these debug statements instead of console statements?

Reply:
This is just for tests, not part of the lib.